content/docs/reference-dom-elements.md

```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
 }
 ```
 
-Note that styles are not autoprefixed. To support older browsers, you need to supply corresponding style properties:
+注意 style 不會自動加上前綴。為了要支援舊的瀏覽器，你必須自行提供相對應的 style property。
 
 ```js
 const divStyle = {
-  WebkitTransition: 'all', // note the capital 'W' here
-  msTransition: 'all' // 'ms' is the only lowercase vendor prefix
+  WebkitTransition: 'all', // 注意這裡是大寫的「W」
```

Comment:
We don't need translate code comments, thank you.

Reply:
I think we should translate comments according to this [discussion] and other merged PRs. (https://github.com/reactjs/zh-hant.reactjs.org/issues/1#issuecomment-465361997)